recipes/intake-astro/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "0.1.1" %}
+{% set name = "intake-astro" %}
+{% set sha256 = "8f2a5f49a10a52cd58770154075de27229242e9ae5c4e95c6a449b285653b90d" %}
+
+package:
+   name: {{ name }}
+   version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+    - jinja2
+  run:
+    - python
```

Comment:
Do you need `python >=3.5` only to build but it runs on `2.7`? If not you need to use `>=3.5` here too.

Reply:
For the moment, Intake itself only runs on python 3.5+, so it's implicit, but that may change. There is nothing non-py2 in this repo, so I'm not sure.

Reply:
In that case I would constrain the plugin to `>=3.5` too. I'm not sure what are the side effects of letting the plugin be installable on py2.7 when a dependency is not: unsatisfied deps or Python upgrade? One is OK the other may surprise users.

Reply:
done